var/macos/install.md

```
@@ -0,0 +1,383 @@
+
+## 5. Run OliveTin
+
+From the folder that contains both `OliveTin` and `config.yaml`:
+
+```sh
+./OliveTin
+```
+
+Then open the web interface at:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)